ajax/libs/pixi.js/package.json

```diff
@@ -15,10 +15,12 @@
   "npmName": "pixi.js",
   "npmFileMap": [
     {
-      "basePath": "bin",
+      "basePath": "dist",
       "files": [
         "pixi.js",
-        "pixi.dev.js"
+        "pixi.js.map",
+        "pixi.min.js",
+        "pixi.min.js.map"
```

Comment:
We could just use single `"**/*"` here to grab all the files, would you please use this rule to replace all the others? Thanks.